theforeman.org/scripts/debian/configure_pbuilder.sh

```diff
@@ -5,28 +5,14 @@ set -xe
 echo "--Apt Hook Setup"
 
 pbuilder_hooksdir=/etc/pbuilder/${os}64/hooks
+local_hooksdir=$(PWD)/hooks-${os}-${version}-${BUILD_ID}
 
-# Cleanup any old hooks
-sudo rm -f ${pbuilder_hooksdir}/F60addforemanrepo
+cp --recursive --preserve=mode ${pbuilder_hooksdir} ${local_hooksdir}
 
-if [[ $repoowner != theforeman ]]; then
-  # Add a stagingdeb repo to the sources too, if it exists
-  if wget -O/dev/null -q http://stagingdeb.theforeman.org/dists/${os}/${repoowner}-${version} ; then
-    echo "echo deb http://stagingdeb.theforeman.org/ ${os} ${repoowner}-${version} >> /etc/apt/sources.list" | sudo tee -a ${pbuilder_hooksdir}/F60addforemanrepo > /dev/null
-  fi
-  if wget -O/dev/null -q http://stagingdeb.theforeman.org/dists/plugins/${repoowner} ; then
-    echo "echo deb http://stagingdeb.theforeman.org/ plugins ${repoowner} >> /etc/apt/sources.list" | sudo tee -a ${pbuilder_hooksdir}/F60addforemanrepo > /dev/null
-  fi
-fi
```

Comment:
as discussed, this is gone.

one thing that I wonder tho, this didn't add the staging repo for `theforeman` builds, which means we couldn't ever build against updated dependencies without a pipeline (which could possibly fail?!) that sounds wrong to me.

Reply:
Indeed, then it would be different from Koji where we do update the buildroots with newly built packages. We should continue to include staging. I'd also expect that it is especially needed for `theforeman`. Perhaps that's a regression we haven't dealt with yet. Maybe we haven't seen it yet since we don't have a lot of build dependencies on our own packages and need that even less.

Reply:
indeed. this is a regression in 08820f20a781886c54bc50bf035ba029de25e995

`configure_pbuilder` used to add the `$repoowner` repo  unconditionally, and only `configure_pbuilder_plugin` had the `if`

Reply:
added back in simplified form